packages/core/src/amazonqTest/chat/controller/controller.ts

```diff
@@ -724,7 +722,7 @@ export class TestController {
             acceptedCharactersCount: session.charsOfCodeAccepted,
             generatedLinesCount: session.linesOfCodeGenerated,
             acceptedLinesCount: session.linesOfCodeAccepted,
-            cwsprChatProgrammingLanguage: session.fileLanguage,
+            cwsprChatProgrammingLanguage: session.fileLanguage ?? 'plaintext',
```

Comment:
Is this the proper place to put this default? Will other functions also default to plaintext? If so, initialize in a common place.

Reply:
@chungjac is working on this.
We aim to declare the telemetry function in telemetryHelper.ts and use it here but for now fixing this here.